rust/src/lib/net_state.rs

```diff
@@ -212,6 +214,18 @@ impl NetworkState {
         self
     }
 
+    /// Nmstate by default will merge information including `ipv4`,
+    /// `ipv6`, `routes`, `route-rules` and `ovn` sections.
+    /// When set to true, the network state be applied without merging current
+    /// network state. For example, if desired state has no IP section defined,
+    /// nmstate will treat it as disabled.
+    /// When set to true, the `state: absent` for routes and route rules will
+    /// not be take effect.
```

Comment:
Why `state: absent` won't take effect? As I understand what this is doing, it will remove routes and rules for the interfaces mentioned in the desired state. But if routes for other interfaces are mentioned as `absent`, why not remove them?

Reply:
The `state: absent` is partial editing, it should not exist in overriding mode.

Meanwhile, the `state: absent` could modify interface not mentioned in desired state which might not expected by user. So for now, I think intend to leave it as not supported and wait user request on this.